moskito-frontend/src/components/Navigation/Navigation.js

```diff
@@ -61,4 +69,17 @@ const NavListItem = styled.li`
         height: 2em;
         stroke: #c97f63;
     }
-`  
\ No newline at end of file
+    button svg{
+        width: 2.3em;
+        height: 2.3em;
+    }
+
+    &:hover,
+    &:active,
+    &:focus {
+        background: linear-gradient(145deg, #A2B2B0, #E2FAF6);
```

Comment:
There is a view that color variables should be used where possible. (I know it's hard)